### ISSUE TITLE
Segmentation fault running CaPTk on the cluster

### ISSUE THREAD

**Issue by waltman** [CBICA/CaPTk#875]
**Describe the bug**
When I run captk on the cluster I get a segmentation fault

**To Reproduce**
```
$ ssh -Y cbica-cluster
$ module load captk/1.7.6
$ captk
Checking for compatible OpenGL - this will happen only once.
QOpenGLFunctions created with non-current context
Segmentation fault
$
```

**Expected behavior**
I was expecting to see the CaPTk GUI.

**Screenshots**
N/A

**CaPTk Version**
1.7.6

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version: Ubuntu 18.04.4 LTS

**Additional context**
N/A

**Comment by ashishsingh18** [comment]
related to #831, #874

**Comment by dboun** [comment]
Duplicate of #831
